src/modules/session-recorder/SessionRecorder.ts

```diff
@@ -104,16 +104,15 @@ export class SessionRecorder {
       checkoutEveryNms: this.#options.maxDuration,
       emit: (event, isCheckout) => {
         if (isCheckout) {
-          let previousSessionEvents = this.#lastTwoSessionEvents[1];
-          const previousSessionRRWebEvents = previousSessionEvents[RQSessionEventType.RRWEB];
-          const timeElapsedSinceStart = event.timestamp - previousSessionRRWebEvents[0].timestamp;
-
+          const previousSessionEvents = this.#lastTwoSessionEvents[1];
+          const previousSessionRRWebEvents = previousSessionEvents[exports.RQSessionEventType.RRWEB];
+          const timeElapsedInBucket =
+            previousSessionRRWebEvents[previousSessionRRWebEvents.length - 1].timestamp -
```

Comment:
you should first check if `previousSessionRRWebEvents.length > 1`, otherwise do nothing.